docs/components/tasklist/introduction-to-tasklist.md

```diff
@@ -20,3 +20,8 @@ we recommend using the REST API to ensure long-term compatibility.
 :::
 
 Tasklist is also available for production use (with support) in the Camunda 8 offering. To try out Tasklist in Camunda 8, sign up [here](https://signup.camunda.com/accounts?utm_source=docs.camunda.io&utm_medium=referral).
+
+<!--- In the Operate overview, we not availability based on trial/professional use, is that relevant for Tasklist as well then? See below:
```

Comment:
@johanwelgemoed This is the final comment I have -- could you clarify for me?

Reply:
This is not relevant for Tasklist. 

Also, I'm not seeing that link on the overall terms list. @christinaausley can you follow up internally with legal to see if this trial license terminology is still valid or should be replaced with a different link?